src/EventFeed.AspNetCore/EventFeedMiddleware.cs

```diff
@@ -150,7 +171,7 @@ private async Task EventFeedPage(HttpContext context, string basePath, int event
             var isComplete = events.Length == eventsPerPage;
             var content = new EventFeedPage(pageNumber, events, links, isComplete);
             context.Response.ContentType = "application/json";
-            await context.Response.WriteAsync(JsonSerializer.Serialize(content));
+            await context.Response.WriteAsync(JsonSerializer.Serialize(content, EventFeedPageSerializerContext.Default.EventFeedPage));
```

Comment:
Maybe we can pull out a `Serialize` method that takes content and requires this context? It is optional but now required for the serialization to be correct, right?

Reply:
If the context is required now for serialization to be correct, this is another cost of idiomatic.